umbraco/Umbraco-CMS
# Add "Move" option for deleted items in the media tree

### Prerequisites


### Description

This PR is the media tree equivalent to #3772. It adds move capability inline in the restore dialog if an item in the media tree can't be restored automatically to its original location.

### Testing this PR

1. Delete a folder with some items in it.
2. Attempt to restore one of the items from the recycle bin.
3. Verify that you're prompted to move it manually and that you can do so directly in the restore dialog.
4. Attempt to restore the folder.
5. Verify that the folder can be restored automatically to its original location in the tree.

It looks like this:

![media-restore-move](https://user-images.githubusercontent.com/7405322/50269215-f01d4400-042d-11e9-8412-12d56a6a1881.gif)

🎄 20/24 🎄
